### PR TITLE
README: highlight a few features

### PR DESCRIPTION
**Pull request by woodruffw**
Sell ourselves a bit!

We do this for most of our other OSS projects, so I figure it doesn't hurt here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,15 @@ of vulnerability reports.
 This project is developed by [Trail of Bits](https://www.trailofbits.com/) with
 support from Google. This is not an official Google product.
 
+## Features
+
+* Support for auditing both local environments and requirements-style files
+* Support for multiple vulnerability services (PyPI, OSV)
```
Comment (di) [inline README.md:19]:
```suggestion
* Support for multiple vulnerability services ([PyPI](https://warehouse.pypa.io/api-reference/json.html#known-vulnerabilities), [OSV](https://osv.dev/docs/))
```

File: README.md [hunk 2]
```diff
@@ -13,6 +13,15 @@ of vulnerability reports.
...
+* Support for emitting SBOMs as CycloneDX XML or JSON
```
Comment (di) [inline README.md:20]:
```suggestion
* Support for emitting [SBOMs](https://en.wikipedia.org/wiki/Software_bill_of_materials) as CycloneDX XML or JSON
```

File: README.md [hunk 3]
```diff
@@ -13,6 +13,15 @@ of vulnerability reports.
...
+* Human and machine-readable output formats (columnar, JSON)
+* "Strict" and "dry-run" modes for high assurance and evaluation
```
Comment (di) [inline README.md:22]:
```suggestion
```
Not sure we're totally ready to use the words 'high assurance' in the context of this project 😉

File: README.md [hunk 4]
```diff
@@ -13,6 +13,15 @@ of vulnerability reports.
...
+* Seamlessly reuses your local `pip` caches
```
Comment (di) [inline README.md:23]:
```suggestion
* Seamlessly reuses your existing local `pip` caches
```

File: README.md (hunk 3 above)
Comment (woodruffw) [reply README.md:22]: Fair!

**Review by di — APPROVED** [review]

**Pull request merged**